packages/backend-function/src/factory.ts

```diff
@@ -305,6 +322,22 @@ class AmplifyFunction
       });
     },
   });
+
+  /**
+   * Store storage outputs using provided strategy
+   */
+  private storeOutput = (
+    outputStorageStrategy: BackendOutputStorageStrategy<FunctionOutput> = new StackMetadataBackendOutputStorageStrategy(
```

Comment:
I don't think we need this default because this construct doesn't have to set output outside of `defineBackend`, which will always include this input